src/com/sun/ts/tests/servlet/spec/errorpage/URLClient.java

```diff
@@ -146,7 +146,7 @@ public void wrappedExceptionTest() throws Fault {
     TEST_PROPS.setProperty(SEARCH_STRING, "Second ErrorPage|"
         + "Servlet Name: WrappedException|"
         + "Request URI: /servlet_spec_errorpage_web/WrappedException|Status Code: 500|"
-        + "Exception Type: null|"
```

Comment:
Why are we removing null?

Reply:
I think that I understand now, the null was incorrectly checked for (due to the attribute name typo that is also being fixed by this pr).